cosmwasm/contracts/pyth/src/msg.rs

```diff
@@ -12,17 +12,26 @@ type HumanAddr = String;
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct InstantiateMsg {
-    pub wormhole_contract:  HumanAddr,
-    pub pyth_emitter:       Binary,
-    pub pyth_emitter_chain: u16,
+    pub wormhole_contract:          HumanAddr,
+    pub pyth_emitter:               Binary,
```

Comment:
Not related to this PR, but can we change the Inst..Msg to get a list of emitter addresses and chains instead of 1?